docs/current_docs/manuals/developer/cookbook/snippets/secret-file/typescript/index.ts

```diff
@@ -0,0 +1,23 @@
+import { dag, object, func, Secret } from "@dagger.io/dagger"
+
+@object()
+class MyModule {
+  /**
+   * Query the GitHub API
+   */
+  @func()
+  async githubApi(
+    /**
+     * GitHub Hosts configuration File
+     */
+    file: Secret,
+  ): Promise<string> {
+    return await dag
+      .container()
+      .from("alpine:3.17")
+      .withExec(["apk", "add", "github-cli"])
+      .withSecretVariable("/root/.config/gh/hosts.yml", file)
```

Comment:
```suggestion
      .withMountedSecret("/root/.config/gh/hosts.yml", file)
```
should be mounted secret file and not env var